bundle/manifests/numaresources-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,284 @@
+          resources:
+          - clusterversionss
+          verbs:
+          - list
+        - apiGroups:
+          - nodetopology.openshift.io
+          resources:
+          - numaresourcesoperators
+          verbs:
+          - create
+          - delete
+          - get
+          - list
+          - patch
+          - update
+          - watch
+        - apiGroups:
+          - nodetopology.openshift.io
+          resources:
+          - numaresourcesoperators/finalizers
+          verbs:
+          - update
+        - apiGroups:
+          - nodetopology.openshift.io
+          resources:
+          - numaresourcesoperators/status
+          verbs:
+          - get
+          - patch
+          - update
+        - apiGroups:
+          - rbac.authorization.k8s.io
+          resources:
+          - rolebindings
+          verbs:
+          - create
+          - delete
+          - get
+          - list
+          - patch
+          - update
+          - watch
+        - apiGroups:
+          - rbac.authorization.k8s.io
+          resources:
+          - roles
+          verbs:
+          - create
+          - delete
+          - get
+          - list
+          - patch
+          - update
+          - watch
+        - apiGroups:
+          - topology.node.k8s.io
+          resources:
+          - noderesourcetopologies
+          verbs:
+          - create
+          - get
+          - list
+          - update
+        - apiGroups:
+          - authentication.k8s.io
+          resources:
+          - tokenreviews
+          verbs:
+          - create
+        - apiGroups:
+          - authorization.k8s.io
+          resources:
+          - subjectaccessreviews
+          verbs:
+          - create
+        serviceAccountName: numaresources-controller-manager
+      deployments:
+      - name: numaresources-controller-manager
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              control-plane: controller-manager
+          strategy: {}
+          template:
+            metadata:
+              labels:
+                control-plane: controller-manager
+            spec:
+              containers:
+              - args:
+                - --secure-listen-address=0.0.0.0:8443
+                - --upstream=http://127.0.0.1:8080/
+                - --logtostderr=true
+                - --v=10
+                image: gcr.io/kubebuilder/kube-rbac-proxy:v0.8.0
+                name: kube-rbac-proxy
+                ports:
+                - containerPort: 8443
+                  name: https
+                  protocol: TCP
+                resources: {}
+              - args:
+                - --platform=kubernetes
+                - --health-probe-bind-address=:8081
+                - --metrics-bind-address=127.0.0.1:8080
+                - --leader-elect
+                command:
+                - /bin/numaresources-operator
+                image: quay.io/fromani/rte-operator:latest
```

Comment:
this needs to be updated

Reply:
quay.io/fromani/numaresources-operator:v4.10.0 ?

Reply:
`quay.io/openshift-kni/numaresources-operator:v4.10.0`

Reply:
> `quay.io/openshift-kni/numaresources-operator:v4.10.0`

correcting myself, looking at the tags prow generates (https://quay.io/repository/openshift-kni/performance-addon-operator?tab=tags) it should be

`quay.io/openshift-kni/numaresources-operator:4.10-snapshot`